### DISCUSSION TITLE
4/4(화) TodoList 만들기

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#54] [General]
물론 전 7월에 아주 늦으으으게 했지만 우리 후배는 너무 잘하기 때문에 swift로 todolist 만들어 오면 되겠습니다.
조건은 navigationItem 을 사용해보기 UserDefult를 UITableView 써보기 TodoList 만들기 지우기 기능과 순서 바꾸기 기능을 구현하기 그리고 아럿을 사용해주세요

참고로 tablewView에 대해서 지식이 있어야하고 navigation에 대한 이해도가 있으면 좋을 것 같습니다!!! 😈

**Comment by jjunhaa0211** [comment]
검사를 위해서 프로젝트 파일은 Smail Info에 올려주세여

**Comment by Eunho0922** [comment]
넹
